src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/HearingDuration.java

```diff
@@ -0,0 +1,725 @@
+    DURATION_088_ON(88,ON,null,45,75,30),
+    DURATION_088_OS(88,OS,null,60,90,30),
+    DURATION_088_OX(88,OX,null,60,90,30),
+    DURATION_088_PF(88,PF,null,45,75,30),
+    DURATION_088_RA(88,RA,null,45,75,30),
+    DURATION_088_RC(88,RC,null,45,75,30),
+    DURATION_088_SF(88,SF,null,45,75,30),
+    DURATION_088_SM(88,SM,null,45,75,30),
+    DURATION_089_CC(89,CC,null,45,75,30),
+    DURATION_089_CE(89,CE,null,30,60,30),
+    DURATION_089_CP(89,CP,null,45,75,30),
+    DURATION_089_DD(89,DD,null,45,75,30),
+    DURATION_089_EC(89,EC,null,45,75,30),
+    DURATION_089_EX(89,EX,null,45,75,30),
+    DURATION_089_GC(89,GC,null,30,60,30),
+    DURATION_089_LE(89,LE,null,30,60,30),
+    DURATION_089_OC(89,OC,null,60,90,30),
+    DURATION_089_ON(89,ON,null,45,75,30),
+    DURATION_089_OS(89,OS,null,60,90,30),
+    DURATION_089_OX(89,OX,null,60,90,30),
+    DURATION_089_PF(89,PF,null,45,75,30),
+    DURATION_089_RA(89,RA,null,45,75,30),
+    DURATION_089_RC(89,RC,null,45,75,30),
+    DURATION_089_SF(89,SF,null,45,75,30),
+    DURATION_089_SM(89,SM,null,45,75,30),
+    DURATION_090_DD(90,DD,null,45,75,30),
+    DURATION_090_FF(90,FF,null,45,75,30),
+    DURATION_094_CB(94,CB,null,45,75,30),
+    DURATION_094_CC(94,CC,null,45,75,30),
+    DURATION_094_CE(94,CE,null,30,60,30),
+    DURATION_094_CP(94,CP,null,45,75,30),
+    DURATION_094_DD(94,DD,null,45,75,30),
+    DURATION_094_EC(94,EC,null,45,75,30),
+    DURATION_094_EX(94,EX,null,45,75,30),
+    DURATION_094_GC(94,GC,null,30,60,30),
+    DURATION_094_LE(94,LE,null,30,60,30),
+    DURATION_094_LT(94,LT,null,60,90,30),
+    DURATION_094_OC(94,OC,null,60,90,30),
+    DURATION_094_ON(94,ON,null,45,75,30),
+    DURATION_094_OS(94,OS,null,60,90,30),
+    DURATION_094_OX(94,OX,null,60,90,30),
+    DURATION_094_RA(94,RA,null,45,75,30),
+    DURATION_094_RC(94,RC,null,45,75,30),
+    DURATION_095_CB(95,CB,null,45,75,30),
+    DURATION_095_CC(95,CC,null,45,75,30),
+    DURATION_095_CE(95,CE,null,30,60,30),
+    DURATION_095_CP(95,CP,null,45,75,30),
+    DURATION_095_DD(95,DD,null,45,75,30),
+    DURATION_095_LE(95,LE,null,30,60,30),
+    DURATION_095_OS(95,OS,null,60,90,30),
+    DURATION_095_RA(95,RA,null,45,75,30),
+    DURATION_096_CC(96,CC,null,null,null,30),
+    DURATION_096_CE(96,CE,null,30,60,30),
+    DURATION_096_CP(96,CP,null,null,null,30),
+    DURATION_096_CR(96,CR,null,45,75,30),
+    DURATION_096_DD(96,DD,null,45,75,30),
+    DURATION_096_EC(96,EC,null,null,null,30),
+    DURATION_096_EX(96,EX,null,45,75,30),
+    DURATION_096_GC(96,GC,null,30,60,30),
+    DURATION_096_LE(96,LE,null,30,60,30),
+    DURATION_096_OC(96,OC,null,null,null,30),
+    DURATION_096_ON(96,ON,null,45,75,30),
+    DURATION_096_OS(96,OS,null,null,null,30),
+    DURATION_096_OX(96,OX,null,null,null,30),
+    DURATION_096_RA(96,RA,null,45,75,30),
+    DURATION_096_RC(96,RC,null,null,null,30);
+
+    private final int benefitCode;
+    private final Issue issue;
+    private final String element;

```

Comment:
What is an element?

Reply:
Looks like they're all null.. do we actually need this element field?

Reply:
Its removed